packages/engine/src/workerpool/mod.rs

```diff
@@ -280,19 +281,19 @@ impl WorkerPoolController {
                 log::debug!("Received RunnerErrors Message from Worker");
                 self.top_send
                     .inner
-                    .send((None, WorkerPoolToExpCtlMsg::Errors(errors)))?;
+                    .send((Some(sim_id), WorkerPoolToExpCtlMsg::Errors(errors)))?;
```

Comment:
Is the simulation id still required to be an `Option`?

Reply:
It would seem not. I think the reasoning for the Option originally is that not all the messages might be related to simulations but instead just something to do with the WorkerPool, but currently that's not the case so will remove the Option